demo/examples/toggle-box.jsx

```diff
@@ -87,7 +87,7 @@ var ToggleBox = React.createClass({
     return (
       <div className="flex-box flex-column flex-1 align-items-center">
         <div>
-          <select rvalue={this.state.effect} onChange={this.whenSelectChanged}>{this.renderEffects()}</select>
```

Comment:
Wondering what the intent of this property was, can't find information on it anywhere. Plus it through an "Unknown property" warning (same thing down below in trigger-box).